src/components/Footer.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="footer__container d-flex flex-row justify-content-end mx-5 mt-3">
+    <div class="footer-items__container d-flex flex-row my-auto mx-3">
+      <a class="footer__item d-flex flex-column align-items-center justify-items-center" ref="telegram" data-bs-toggle="tooltip" data-bs-placement="top" data-bs-title="Telegram" href="https://t.me/dnjco" target="_blank">
+        <img class="m-auto" src="@/assets/icons/telegram.svg" alt="">
```

Comment:
I already asking you to change 4 images to font icons, should you ask you for every instance?
IF YOU CAN USE FONT ICONS, USE IT !!!
We living in iran, our customers and users living in Iran, network connection is a catastrophe, every http request took about 100-200 ms!

Reply:
telegram icon was [removed](https://github.com/Templarian/MaterialDesign/issues/5275) from @mdi/fonts so i decided to use files instead to match exactly like UI.
But now i used mdi-send instead